internal/conditions/conditions.go

```diff
@@ -105,6 +105,7 @@ var fluentBitLogPipelineMessages = map[string]string{
 
 var otelLogPipelineMessages = map[string]string{
 	ReasonEndpointInvalid: "OTLP output endpoint invalid: %s",
+	ReasonOTTLSpecInvalid: "OTTL specification is invalid, %s. Fix the syntax error indicated by the message or see troubleshooting: https://kyma-project.io/#/telemetry-manager/user/02-logs?id=ottl_spec_invalid_with_unspecific_error_message",
```

Comment:
The three added links are all invalid, please link properly. ALso see the newly added constants for links on top of the file.